### PR TITLE
Added ui part

### PR DESCRIPTION
**Pull request by Nuta88**

### CODE REVIEW

File: .idea/happy-pig.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (AlexKruhlov) [inline .idea/happy-pig.iml]: Please add the empty new line

File: ui/angular.json [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      }
+    }
+  },
+  "defaultProject": "ui",
+  "schematics": {
+    "@schematics/angular:component": {
+      "styleext": "scss"
+    }
+  }
+}
```
Comment (AlexKruhlov) [inline ui/angular.json]: Please add the empty new line

File: ui/e2e/protractor.conf.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    defaultTimeoutInterval: 30000,
+    print: function() {}
+  },
+  onPrepare() {
+    require('ts-node').register({
+      project: require('path').join(__dirname, './tsconfig.e2e.json')
+    });
+    jasmine.getEnv().addReporter(new SpecReporter({ spec: { displayStacktrace: true } }));
+  }
+};
```
Comment (AlexKruhlov) [inline ui/e2e/protractor.conf.js]: Please add the empty new line

File: ui/e2e/tsconfig.e2e.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    "outDir": "../out-tsc/app",
+    "module": "commonjs",
+    "target": "es5",
+    "types": [
+      "jasmine",
+      "jasminewd2",
+      "node"
+    ]
+  }
+}
```
Comment (AlexKruhlov) [inline ui/e2e/tsconfig.e2e.json]: Please add the empty new line

File: ui/src/app/main/app.component.html [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<!--The content below is only a placeholder and can be replaced.-->
+<div>
+<router-outlet></router-outlet>
+</div>
+
+
```
Comment (AlexKruhlov) [inline ui/src/app/main/app.component.html]: Please remove the redundant line

File: ui/src/app/service/fund.service.ts [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+import { HttpClient } from '@angular/common/http';
+
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class FundService {
+
+  private url = 'http://localhost:8080';
```
Comment (AlexKruhlov) [inline ui/src/app/service/fund.service.ts]: For the future you should think about moving main url to somthing common place.

File: ui/src/browserslist [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# This file is currently used by autoprefixer to adjust CSS to support the below specified browsers
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+# For IE 9-11 support, please uncomment the last line of the file and adjust as needed
+> 0.5%
+last 2 versions
+Firefox ESR
+not dead
+# IE 9-11
```
Comment (AlexKruhlov) [inline ui/src/browserslist]: Please add the empty new line

File: ui/src/karma.conf.js [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+    },
+    reporters: ['progress', 'kjhtml'],
+    port: 9876,
+    colors: true,
+    logLevel: config.LOG_INFO,
+    autoWatch: true,
+    browsers: ['Chrome'],
+    singleRun: false
+  });
+};
```
Comment (AlexKruhlov) [inline ui/src/karma.conf.js]: Please add the empty new line

File: ui/angular.json (hunk 2 above)
Comment (Nuta88) [reply ui/angular.json]: done

File: ui/e2e/protractor.conf.js (hunk 3 above)
Comment (Nuta88) [reply ui/e2e/protractor.conf.js]: done

File: ui/e2e/tsconfig.e2e.json (hunk 4 above)
Comment (Nuta88) [reply ui/e2e/tsconfig.e2e.json]: done

File: ui/src/app/service/fund.service.ts (hunk 6 above)
Comment (Nuta88) [reply ui/src/app/service/fund.service.ts]: ok

File: ui/src/karma.conf.js (hunk 8 above)
Comment (Nuta88) [reply ui/src/karma.conf.js]: done

File: ui/src/browserslist (hunk 7 above)
Comment (Nuta88) [reply ui/src/browserslist]: done

**Pull request merged**